### DISCUSSION TITLE
Test dockerized web app on PC browser

### DISCUSSION THREAD

**Discussion by Padmanabhan100** [docker/extension-ideas#181] [Ideas]
When we run the app(flask, django or any other app) we must be able to access it on our pc via a link to test it.

**Comment by haticecan37** [comment]
```
helm repo add kubescape https://kubescape.github.io/helm-charts/ ; 
helm repo update ; 
helm upgrade --install kubescape kubescape/kubescape-operator -n kubescape --create-namespace --set clusterName=`kubectl config current-context` --set account=df4c2e83-3740-4d79-9b50-67fba02f46f6 --set server=api.armosec.io
```
![vitest](https://github.com/docker/extension-ideas/assets/140960306/760a2f16-fdea-4c85-b63d-a8d68243bb8a)

**Comment by haticecan37** [comment]
helm repo add kubescape https://kubescape.github.io/helm-charts/ ; 
helm repo update ; 
helm upgrade --install kubescape kubescape/kubescape-operator -n kubescape --create-namespace --set clusterName=`kubectl config current-context` --set account=df4c2e83-3740-4d79-9b50-67fba02f46f6 --set server=api.armosec.io[https://github.com/docker/extension-ideas/discussions/181#discussioncomment-7154192
![Uploading IMG_0552.png…]()
](url)
